src/views/BreadView/BreadTable.js

```diff
@@ -0,0 +1,178 @@
+import * as api from '../../api';
+import * as utils from '../../utils';
+
+const emptyRow = (classes, message) => {
+    return (
+        <TableRow>
+          <TableCell className={classes.dateCell}>{message}</TableCell>
+          <TableCell className={classes.descriptionCell}></TableCell>
+          <TableCell className={classes.buttonCell}>
+            <IconButton title="Lock" disabled>
+              <LockIcon fontSize="small" />
+            </IconButton>
+          </TableCell>
+        </TableRow>
+    )
+};
+
+
+class BreadTable extends React.Component {
+
+  state = {
+    dialogOpen: false,
+    submitting: false,
+    order: {},
+    items: {},
+    selectedItem: "buh"
+  }
+
+  handleAddClick = (order) => () => {
+    this.setState({dialogOpen: true, order: order});
+  }
+
+  handleClearOrder = (order) => () => {
+      this.setState(
+          {dialogOpen: false, order: order},
+          () => this.doClear()
+      );
+  }
+
+  doClear = () => {
+    api.del({
+        path: '/bread/' + this.state.order.id,
+    }).then(data => {
+        this.setState(
+          {submitting: false, order: {}},
+          () => this.props.showMessage('Bestelling verwijderd', this.props.refresh)
+        );
+    }).catch(error => {
+        this.setState(
+          {submitting: false, order: {}},
+          () => this.props.showMessage(error.message)
+        );
+    })
+  }
+
+  handleAddToOrder = value => {
+    this.setState(
+        {dialogOpen: false, submitting: true},
+        () => this.doAddItem(value)
+    );
+  }
+
+  doAddItem = value => {
+    api.patch({
+        path: '/bread/' + this.state.order.id,
+        data: { items: [value] },
+    }).then(data => {
+        this.setState(
+          {submitting: false, order: {}},
+          () => this.props.showMessage('Brood toegevoegd', this.props.refresh)
+        );
+    }).catch(error => {
+        this.setState(
+          {submitting: false, order: {}},
+          () => this.props.showMessage(error.message)
+        );
+    })
+  }
+
+  handleDialogChange = (dialogOpen) => () => {
+    this.setState({dialogOpen: dialogOpen});
+  }
+
+  render () {
+    const { classes, orders, loading, items } = this.props;
+
+    return (
+        <React.Fragment>
+          <Table>
+            <TableHead>
+              <TableRow>
+                <TableCell className={classes.dateCell}>Week</TableCell>
+                <TableCell className={classes.ordersCell}>Bestelling</TableCell>
+                <TableCell className={classes.buttonCell}></TableCell>
+              </TableRow>
+            </TableHead>
+            <TableBody>
+              {loading ? emptyRow(classes) : null}
+              {!loading && orders.length === 0 ? emptyRow(classes, 'Geen bestellingen mogelijk') : null}
+              {orders.map(row => {
+                 return (
+                   <TableRow key={row.id}>
+                     <TableCell className={classes.dateCell}>
```

Comment:
brood actie icoontjes naar voor of starten met een brood icoon/avatar?

Reply:
Voor zover ik zie blijven die knoppen altijd in zicht, dus dit lijkt me logischer

Reply:
ook op mobile?